Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -92,6 +92,11 @@ public class NetworkManager : MonoBehaviour
         [FormerlySerializedAs("m_MaxConnections")]
         public int maxConnections = 4;
 
+        [Header("Authentication")]
+
+        [SerializeField]
+        protected NetworkAuthenticator authenticator;
```

Comment:
you could make this public for consistency with the rest of the variables, and for easier access